docs/source/download.rst

```diff
@@ -1,111 +1,200 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are two main approaches to installing Devito.
+
+- `Docker installation`_, for those looking for a quick “out-of-the-box” way to try Devito
```

Comment:
old sentence sounded better IMHO

", for those looking for the least-friction way to try Devito"

Reply:
ok